tests/conftest.py

```diff
@@ -364,25 +366,67 @@ def pytest_generate_tests(metafunc):
         metafunc.parametrize("test_backends", [test_backends], scope="module")
 
 
+@dataclass(frozen=True)
+class TestMarkerCoverage:
+    path: str
+    name: str
+    markers: set[str]
+
+    def __str__(self):
+        return f"{self.path}, {self.name}, {self.markers}"
+
+
 def _verify_marker_coverage(
     session,
-) -> list[tuple[str, str, list[str]]]:
-    uncovered: list[tuple[str, str, set[str]]] = []
+) -> tuple[list[TestMarkerCoverage], list[TestMarkerCoverage]]:
+    uncovered: list[TestMarkerCoverage] = []
+    multiple_markers: list[TestMarkerCoverage] = []
     for test in session.items:
         markers = {m.name for m in test.iter_markers()}
-        if not REQUIRED_MARKERS.intersection(markers):
-            uncovered.append((str(test.path), test.name, markers))
-    return uncovered
+        required_intersection = markers.intersection(REQUIRED_MARKERS)
```

Comment:
This is the new logic for testing only 1 marker is present.

Reply:
:rocket: